src/exposurelog/routers/add_message.py

```diff
@@ -128,44 +126,60 @@ async def add_message(
     return Message.from_orm(result)
 
 
-def get_day_obs_from_registries(
+def exposure_from_registry(
     registries: collections.abc.Sequence[lsst.daf.butler.registry.Registry],
     obs_id: str,
     instrument: str,
-) -> None | int:
+) -> lsst.daf.butler.dimensions.DimensionRecord:
     """Get the day of observation of an exposure, or None if not found.
 
     Parameters
     ----------
     registries
         One or more data registries.
         They are searched in order.
-    obs_id
-        Observation ID.
     instrument
         Instrument name.
+    obs_id
```

Comment:
Why the change of order in the doc string and not in the argument list itself? Also, please add the expected types to the doc strings of all parameters.

Reply:
Good point. I'll change it everywhere.